tests/functional/conftest.py

```diff
@@ -63,6 +64,14 @@ def pytest_sessionstart(session):
     dr.Client(endpoint=ENDPOINT_URL, token=os.environ["DATAROBOT_API_TOKEN"])
 
 
+@pytest.fixture(scope="session")
+def uwsgi_env():
```

Comment:
Where is the uwsgi config itself? I couldnt find anything in the PR

Reply:
https://github.com/datarobot/mlpiper/blob/master/mlpiper-py/mlpiper/components/restful/uwsgi_ini_template.py